lib/sass.js

```diff
@@ -58,12 +57,19 @@ export async function buildCSS(inputs, { tailwindConfig } = {}) {
         to: output,
         map: {
           inline: false,
-          prev: sassResult.map?.toString(),
+          prev: JSON.stringify(sassResult.sourceMap),
         },
       });
+      const sourceMappingURL = relative(dirname(output), sourceMapPath);
 
-      await writeFile(output, postcssResult.css);
-      await writeFile(sourcemapPath, postcssResult.map.toString());
+      await writeFile(
+        output,
+        // We have to manually add the sourceMappingURL comment, because
+        // `sass.compile(...)` does not do it by itself.
+        // https://sass-lang.com/documentation/js-api/interfaces/Options#sourceMap
+        `${postcssResult.css}\n/*# sourceMappingURL=${sourceMappingURL} */`
```

Comment:
I don't think it will make a difference with any of the file names we currently use, but I will note that if we used characters outside the `[0-9a-zA-Z-_.]` set in file names, it may be necessary to percent-encode this (eg. using `encodeURI`).